### ISSUE TITLE
2D radiosity

### ISSUE THREAD

**Issue by knightcrawler25** [diwi/PixelFlow#20]
This is such amazing library. Lots of examples to learn from. Thanks a lot for this. Also, I was going through your website and noticed the 2D radiosity renderer that you've made. Would you be releasing that in the near future? I really would love to see the code behind that. :)

**Comment by sulram** [comment]
+1 for 2D radiosity examples ! Awesome work!

**Comment by noname-2** [comment]
I agree, it would be great to get some 2D radiosity examples. The images look just amazing!

**Comment by NAHNAH55** [comment]
Updates on this?

**Comment by hexus** [comment]
+1, though it's a bit worrying that @diwi's website is down and there's been no GitHub activity since January 28th.

**Comment by hexus** [comment]
A couple of years later and I'm still wondering whether he will return!

**Comment by Krzycho666** [comment]
+1
I look forward for this as well.
